include/itkVariationalRegistrationStopCriterion.hxx

```diff
@@ -86,7 +86,7 @@ VariationalRegistrationStopCriterion< TRegistrationFilter, TMRFilter >
 template< class TRegistrationFilter, class TMRFilter >
 void
 VariationalRegistrationStopCriterion< TRegistrationFilter, TMRFilter >
-::Execute( const itk::Object */*caller*/, const itk::EventObject & /*event*/ )
+::Execute( const itk::Object * /*caller*/, const itk::EventObject & /*event*/ )
```

Comment:
An improvement for compilers and semantic code is to use the `itkNotUsed` macro.